server/spawn-vehicle.lua

```diff
@@ -8,10 +8,15 @@ end
 ---@param source number
 ---@param vehicleId integer
 ---@param garageName string
----@param accessPoint integer
+---@param accessPointIndex integer
 ---@return number? netId
-lib.callback.register('qbx_garages:server:spawnVehicle', function (source, vehicleId, garageName, accessPoint)
+lib.callback.register('qbx_garages:server:spawnVehicle', function (source, vehicleId, garageName, accessPointIndex)
     local garage = Garages[garageName]
+    local accessPoint = garage.accessPoints[accessPointIndex]
+    if #(GetEntityCoords(GetPlayerPed(source)) - accessPoint.coords) > 3 then
```

Comment:
Should probably move this magic number to the config. some folks may want more especially for planes/boats or giant vehicles.